windows-release/stage-layout-embed.yml

```
@@ -46,6 +46,28 @@ jobs:
       --preset-embed
     displayName: 'Generate embeddable layout'
 
+  - powershell: >
```

```suggestion
  - powershell: |
```

Separate commands, so we need them to keep the line breaks.